OutOfSchool/OutOfSchool.IdentityServer/Controllers/AccountController.cs

```diff
@@ -196,6 +196,33 @@ public async Task<IActionResult> SendConfirmEmail()
         return Ok();
     }
 
+    [HttpGet]
+    public async Task<IActionResult> ReSendEmailConfirmation()
+    {
+        var user = await userManager.FindByEmailAsync(User.Identity.Name);
+
+        var token = await userManager.GenerateEmailConfirmationTokenAsync(user);
+        var returnUrl = "Login";
+        var callBackUrl = Url.Action("EmailConfirmation", "Account", new { token, user.Email, returnUrl }, Request.Scheme);
+
+        var email = user.Email;
+        var subject = localizer["Confirm email"];
+        var userActionViewModel = new UserActionViewModel
+        {
+            FirstName = user.FirstName,
+            LastName = user.LastName,
+            ActionUrl = callBackUrl,
+        };
+        var content = await renderer.GetHtmlPlainStringAsync(RazorTemplates.ConfirmEmail, userActionViewModel);
+        await emailSender.SendAsync(email, subject, content);
```

Comment:
`await emailSender.SendAsync(email, subject, content);` - I see no check for unsuccessful send. Just curious - how fails are processed?

Reply:
I've rechecked: in the nested method SendAsync() there is try...catch